### ISSUE TITLE
traceback on wrong path

### ISSUE THREAD

**Issue by RonnyPfannschmidt** [borgbackup/borg#1490]
```
$ borg list --format '{archive_name}{LF}' foo
Exception ignored in: <bound method Repository.__del__ of <Repository /run/media/ronny/backups/foo>>
Traceback (most recent call last):
  File "/nix/store/dv8cnrgx6hxyzcykixk3fzkasf61gfl2-borgbackup-1.0.6/lib/python3.4/site-packages/borg/repository.py", line 123, in __del__
    assert False, "cleanup happened in Repository.__del__"
AssertionError: cleanup happened in Repository.__del__
/run/media/ronny/backups/foo is not a valid repository. Check repo config.
```

**Comment by ThomasWaldmann** [comment]
@enkore ronny was doing that on master, does it apply to 1.0-maint?

**Comment by ThomasWaldmann** [comment]
@RonnyPfannschmidt --short

**Comment by RonnyPfannschmidt** [comment]
whops, there is a pathname missmatch due to not correctly setting some attributes in the nix expression i use to fetch from github

**Comment by ThomasWaldmann** [comment]
@RonnyPfannschmidt so, what does that mean? close?

**Comment by enkore** [comment]
Due to the path mixup it seems that this was master code which indeed has the issue.

**Comment by ThomasWaldmann** [comment]
Fixed by #1496.
